roles/pulp_common/defaults/main.yml

```diff
@@ -6,6 +6,7 @@ pulp_config_dir: '/etc/pulp'
 # foreseeable need for them to change the filename, only pulp_config_dir.
 pulp_settings_file: '{{ pulp_config_dir }}/settings.py'
 pulp_install_source: pip
+pulp_install_object_storage: filesystem
```

Comment:
We could add pretasks with assertions that the selected storage solution is configured.
roughly:
```yaml
- name: Check storage backend is configured.
  assert:
    that:
      - aws_storage_bucket_name is defined
      - <...>
  when: pulp_install_object_storage == "s3"
```